src/test/java/org/example/homework13/BlackJackTests/CardTest.java

```diff
@@ -0,0 +1,40 @@
+package org.example.homework13.BlackJackTests;
+
+import org.example.homework13.Card;
+import org.example.homework13.Rank;
+import org.example.homework13.Suit;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class CardTest {
+
+    @Test
+    @DisplayName("Check getting card suit")
+    @Tag("CardClassTest")
```

Comment:
Tagging Tests with a class does not make big sense. But for learning purpose, it is absolutely ok.
Just for your information, that Junit looks at every Test Class as a separate Test Suite, that's why your tagging will become useless